scripts/scripts.js

```diff
@@ -376,15 +376,14 @@ export async function createCard(row, style) {
  * @param {Object} row JSON Object typically coming from an index array item
  * @param {Array} styles Class names that needs to be added to the card root div
  */
-
 export function createDocumentCard(row, styles) {
   // Create card div
-  const card = document.createElement('div');
+  const card = createTag('div');
   if (styles.length) {
     styles.forEach((style) => card.classList.add(style));
   }
   // Get category
-  const category = document.createElement('div');
+  const category = createTag('div', { class: 'document-category' });
   category.classList.add('document-category');
```

Comment:
You already added the class above in line 386, right?

Reply:
Good catch. I removed line 386.